Taxi/Taxi/Presenter/CellView.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  CellView.swift
+//  Taxi
+//
+//  Created by Yosep on 2022/06/09.
+//
+import SwiftUI
+
+struct CellView: View {
+    var body: some View {
+        VStack(alignment: .leading, spacing: 13) {
+            HStack {
+                MeetingTimeCell()
+                Spacer()
+                CurrentUserCell()
+            }
+            DestinationCell()
+        }
+        .padding(18)
+        .background {
+        ZStack { // Struct로 만들어서 표현하자
+            RoundedRectangle(cornerRadius: 18.0)
+                .fill(Color(.white))
+                .shadow(color: .black.opacity(0.05), radius: 2, x: 0, y: 1)
+            RoundedRectangle(cornerRadius: 16)
+                .stroke(Color(red: 240 / 255, green: 240 / 255, blue: 240 / 255, opacity: 1.0), lineWidth: 1.5)
+        }
+        }
```

Comment:
인덴트가 맞지 않습니다...!
여기뿐 아니라 아래에 있는 background 코드가 전부 이렇네요 수정 부탁드리겠습니다.

```suggestion
        .background {
            ZStack { // Struct로 만들어서 표현하자
                RoundedRectangle(cornerRadius: 18.0)
                    .fill(Color(.white))
                    .shadow(color: .black.opacity(0.05), radius: 2, x: 0, y: 1)
                RoundedRectangle(cornerRadius: 16)
                    .stroke(Color(red: 240 / 255, green: 240 / 255, blue: 240 / 255, opacity: 1.0), lineWidth: 1.5)
            }
        }
```

Reply:
수정완료~!